modules/RTC/TraceablePeerConnection.js

```diff
@@ -969,16 +971,18 @@ TraceablePeerConnection.prototype._remoteTrackAdded = function(stream, track, tr
     const sourceName = this.signalingLayer.getTrackSourceName(trackSsrc);
     const peerMediaInfo = this.signalingLayer.getPeerMediaInfo(ownerEndpointId, mediaType, sourceName);
 
-    if (!peerMediaInfo) {
-        GlobalOnErrorHandler.callErrorHandler(
```

Comment:
Why don't we use the `GlobalOnErrorHandler` anymore?

Reply:
If there is no presence, we don't stop track creation anymore. We go ahead and create the remote track assuming default presence.